client/src/entrypoints/admin/date-time-chooser.js

```diff
@@ -1,4 +1,5 @@
 import $ from 'jquery';
+import { Widget } from './telepath/widgets';
```

Comment:
Importing telepath/widgets and calling `window.telepath.register` from this file will make it fail on any view that doesn't include the main telepath.js module (which currently means any view without a streamfield component). Widget registration code should be in its own file, which is then specified in the Python-side widget adapter's media.